service/application/src/main/java/com/solace/maas/ep/runtime/agent/service/ScanService.java

```diff
@@ -136,10 +146,14 @@ public String singleScan(List<RouteBundle> routeBundles, int numExpectedCompleti
             ScanEntity returnedScanEntity = setupScan(route, routeBundle, savedScanEntity, scanId);
 
             if (!loggingService.hasLoggingProcessor(scanId)) {
+                Map<String, String> details = new HashMap<>();
                 MDC.put(SCAN_ID, scanId);
-                fileLoggerFactory.create();
-                LoggingProcessor loggingProcessor = new LoggingProcessor();
-                loggingService.addLoggingProcessor(scanId, loggingProcessor);
+                details.put(SCAN_ID, scanId);
+                details.put(MESSAGING_SERVICE_ID, routeBundle.getMessagingServiceId());
+                details.put(SCHEDULE_ID, groupId);
+
+                createScanFileLogger(details);
```

Comment:
Why is this still being done here? What happens if we use another method to perform a scan that isn't a single scan. Do we not get logs any more?

Reply:
The coe